typescript/infra/config/environments/testnet2/helloworld/abacus/verification.json

```diff
@@ -1 +1,74 @@
-"{\"alfajores\":[{\"name\":\"router\",\"address\":\"0x0FD5A339466638aD2746748dCfFF65A27f605de4\",\"constructorArguments\":\"000000000000000000000000c41169650335ad274157ea5116cdf227430a68a30000000000000000000000000000000000000000000000000000000000000000\",\"isProxy\":false}],\"kovan\":[{\"name\":\"router\",\"address\":\"0xba838EB8C9850042b4fc8b02b60B14B4cc8945C4\",\"constructorArguments\":\"000000000000000000000000fa1fbf362144ae1bef2e33409948da1fb812bb410000000000000000000000000000000000000000000000000000000000000000\",\"isProxy\":false}],\"fuji\":[{\"name\":\"router\",\"address\":\"0x24F337d760f092624D7DDC1d9B4d8c1A35Fd4F3b\",\"constructorArguments\":\"00000000000000000000000033abaf6708be03bdf0595da0745a7111b01db8c70000000000000000000000000000000000000000000000000000000000000000\",\"isProxy\":false}],\"mumbai\":[{\"name\":\"router\",\"address\":\"0x636bcE43104Ef1E61e93E84F0A324d037C258308\",\"constructorArguments\":\"000000000000000000000000b636b2c65a75d41f0dbe98fb33eb563d245a241a0000000000000000000000000000000000000000000000000000000000000000\",\"isProxy\":false}],\"bsctestnet\":[{\"name\":\"router\",\"address\":\"0x8273C4eeeCd9E0C7A7AD69d07f70C4568321d7c7\",\"constructorArguments\":\"000000000000000000000000e403e16db1f5997bc62dc611a8d42836364a7f010000000000000000000000000000000000000000000000000000000000000000\",\"isProxy\":false}],\"arbitrumrinkeby\":[{\"name\":\"router\",\"address\":\"0xae0189ae2564F0554A783Ac5fc73D2044F4eD984\",\"constructorArguments\":\"000000000000000000000000fb55597f07417b08195ba674f4dd58aec9b89fbb0000000000000000000000000000000000000000000000000000000000000000\",\"isProxy\":false}],\"optimismkovan\":[{\"name\":\"router\",\"address\":\"0xFd8771A559b941B95b66323c890B6fA21f409563\",\"constructorArguments\":\"000000000000000000000000740bed6e4eec7c57a2818177fba3f9e896d5de1c0000000000000000000000000000000000000000000000000000000000000000\",\"isProxy\":false}]}"
+{
```

Comment:
Is the move from string to json object intentional?

Reply:
Yeah it's a result from fixing it in here https://github.com/abacus-network/abacus-monorepo/pull/1047/files#diff-a3af0555ffca8696a2d5ca969345a1cb260c277680e5b3409527537bc43645dfL59